medical_compliance/medical_compliance/api/tasks.py

```diff
@@ -74,6 +74,6 @@ def fetch_heart_rate_measurement():
         )
         heart_rate_measurement.save()
 
-    analyze_heart_rates.delay()
+    analyze_heart_rates.delay(last_meas)
```

Comment:
I generally try to have a clear name for the variables, "no matter" how long they get, so that it's easier to understand what they do, and also keep from shortening words if it's not at all necessary :)